src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte

```diff
@@ -11,7 +11,7 @@
 	$: url = transactionsUrl({ token });
 </script>
 
-<div class="mb-6 flex gap-3 sm:gap-8">
+<div class="group mb-3 flex gap-3 rounded-xl px-3 py-2 hover:bg-white sm:gap-8">
```

Comment:
Does it needs `active:bg-white` as well?

Reply:
for sure it is nice to have, yes, included!